operator/v1/types.go

```diff
@@ -133,6 +133,11 @@ type KlusterletDeployOption struct {
 	// Note: Do not modify the Mode field once it's applied.
 	// +optional
 	Mode InstallMode `json:"mode"`
+
```

Comment:
KlusterletDeployOption is to define whether klusterlet can be run in hosted or default mode. I would think HubApiServerHostAlias should be put in the spec. it cannot be `hub_api_server_host_alias`, it needs to follow the camel format such as "hubAPIServerHostAliase"

Reply:
I had a wrong understanding of `KlusterletDeployOption`. I will put `HubApiServerHostAlias` in the spec and format as `hubAPIServerHostAlias`.